server.js

```diff
@@ -40,6 +43,25 @@ router.post('/text', (req, res)=> {
 })
 
 
+
+
+
+router.get('/api', (req, res) => {
+    console.log("fetching example data with authenticated user in Onet Database")
+
+    fetch('https://services.onetcenter.org/ws/mnm/careers/13-2011.01/skills', { 
+        headers: { 'Authorization': 'Basic c3RpbGxfdW5kZXJfY29uc3RydWM6NzI4MnJjeQ==', 'accept': 'application/json'},
```

Comment:
heads up, usually you'd want to keep a key like this in an .env variable to hide it in a variable in your codebase, I know this is just for example data though